### ISSUE TITLE
Support the new json_mode

### ISSUE THREAD

**Issue by dasheck0** [mudler/LocalAI#1335]
**Is your feature request related to a problem? Please describe.**

I want consistent json outputs of my model especially when providing a json schema. However, most models include an explanation to their responses. Prompt engineering is not enough here.

**Describe the solution you'd like**

OpenAI supports now a json mode that at least guarantees that the output is json. See: https://platform.openai.com/docs/guides/text-generation/json-mode

**Describe alternatives you've considered**

I looked into manually calculating logits. However, seems to be too complicated for me.

**Additional context**

**Comment by mudler** [comment]
This is already supported in master, been introduced with #1254 . Closing
